macros/src/main/scala/packaged/macros2/package.scala

```diff
@@ -0,0 +1,21 @@
+package packaged
+
+import scala.gestalt._
+
+package object macros2 {
+  inline def plus(a: Int, b: Int): Int = meta {
+    q"$a + $b"
+  }
+
+  object PlusObj {
+    inline def plus(a: Int, b: Int): Int = meta {
+      q"$a + $b"
+    }
+  }
+
+  implicit class PlusFor(val a: Int) {
+    inline def plus(b: Int): Int = meta {
+      q"$this.a + $b"
+    }
+  }
```

Comment:
It is for testing macros defined inside of a package object. I couldn't use "macros" because there is already package called that it it did not allow me to add the defs.